### PR TITLE
Fixed Bug in converttopython. 

### PR DESCRIPTION
**Pull request by prvashis**
https://github.com/CiscoUcs/ucsmsdk/issues/54

### CODE REVIEW

File: ucsmsdk/utils/converttopython.py [hunk 1]
```diff
@@ -1219,7 +1219,7 @@ def _generate_cmdlets(xml_string):
     """
 
     doc = xml.dom.minidom.parseString(xml_string)
-
+    top_node = doc.documentElement
```
Comment (vvb) [inline ucsmsdk/utils/converttopython.py]: @prvashis is `top_node` used?

File: ucsmsdk/utils/converttopython.py [hunk 2]
```diff
@@ -1313,7 +1313,9 @@ def _find_xml_requests_in_file(file_stream, gui_log):
             line = file_stream.readline()
             if line is not None:
                 # print "[%s]" %(line)
-                _extract_xml(file_stream, line)
+                if _check_if_any_list_value_in_string(_multi_line_method,
```
Comment (vvb) [inline ucsmsdk/utils/converttopython.py]: @prvashis can you rewrite it as

File: ucsmsdk/utils/converttopython.py (hunk 1 above)
Comment (prvashis) [inline ucsmsdk/utils/converttopython.py]: Extra code...Removed.

File: ucsmsdk/utils/converttopython.py (hunk 2 above)
Comment (prvashis) [inline ucsmsdk/utils/converttopython.py]: Updated the code according to comment.

**Comment by vvb** [comment]
LGTM. can you squash your change to a single commit.

**Pull request merged**